webapp/views/wpt-results.js

```diff
@@ -368,10 +369,25 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     <template is="dom-if" if="[[pathIsATestFile]]">
       <div class="history">
         <template is="dom-if" if="[[!showHistory]]">
-          <paper-button id="show-history" onclick="[[showHistoryClicked()]]" raised>
-            Show history
+          <paper-button id="show-history" onclick="[[showNewHistoryClicked()]]" raised>
+            Show New history
           </paper-button>
+          <paper-button id="show-history" onclick="[[showHistoryClicked()]]" raised>
+          Show history
+        </paper-button>
+        </template>
+        <!-- New Test History Implementation -->
+        <template is="dom-if" if="[[showNewHistory]]">
+        <h3>
+          History <span>(Experimental)</span>
+        </h3>
+        <template is="dom-if" if="[[pathIsATestFile]]">
+          <new-test-results-history-grid
+          data="[[mockData]]">
```

Comment:
I don't see a value for `mockData` here, but also the child component does not use a `data` instance variable. Could this be part of a previous implementation?